elasticsearch/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
```

Comment:
We'll want the description to read "A helm chart for elasticsearch", or some variant